src/libs/tools/src/plugindatabase.cpp

```diff
@@ -223,6 +223,21 @@ std::string ModulesPluginDatabase::lookupInfo (PluginSpec const & spec, std::str
 	return plugin->lookupInfo (which);
 }
 
+PluginDatabase::func_t ModulesPluginDatabase::getSymbol (PluginSpec const & spec, std::string const & which) const
+{
+	PluginPtr plugin;
+	try
+	{
+		plugin = impl->modules.load (spec.getName (), spec.getConfig ());
+	}
+	catch (...)
+	{
+		throw;
+	}
```

Comment:
Yes, I tried to fix #479 which is a real strange error: some exceptions thrown from libtool to kdb are not catched in kdb even though kdb contains a catch-all block. As this try/catch did not help to fix the problem you can safely remove it.